app/src/main/java/com/lagradost/cloudstream3/ui/settings/SettingsGeneral.kt

```diff
@@ -335,6 +335,15 @@ class SettingsGeneral : PreferenceFragmentCompat() {
             return@setOnPreferenceChangeListener true
         }
 
+        settingsManager.edit().putBoolean(
+            getString(R.string.skip_startup_account_select_key),
+            getKey(getString(R.string.skip_startup_account_select_key), false) ?: false
+        ).apply()
+        getPref(R.string.skip_startup_account_select_key)?.setOnPreferenceChangeListener { _, newValue ->
```

Comment:
This entire section can be removed as the it should instead be used like 
```kotlin
val skipStartup = settingsManager.getBoolean(ctx.getString(R.string.skip_startup_account_select_key),false) || accounts.count() <= 1
```

Reply:
jsdelivr_proxy_key is also wrong for the same reason and should be changed too, but you dont need to do that

Reply:
Also this might want to be moved into account section as it is about accounts